modules/sno-du-disabling-ntp.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-disabling-ntp_{context}"]
+= Disabling Network Time Protocol (NTP)
+
+After the system is configured for Precision Time Protocol (PTP), you need to
+remove NTP from impacting the system clock.
+
+.Procedure
+
+* No configuration changes are needed. Use the provided settings:
++
```

Comment:
Does this CR exist or does the user need to create?

Reply:
The user uses the example to create the CR.